mesonbuild/compilers/java.py

```diff
@@ -39,6 +39,9 @@ def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoic
         super().__init__(exelist, version, for_machine, info, full_version=full_version)
         self.javarunner = 'java'
 
+    def get_warn_args(self, level: str) -> T.List[str]:
+        return ['-Xlint:all', '-Xdoclint:all']
```

Comment:
this seems to be ignoring the level parameter entirely?

Reply:
yes. What do you think is the best course of action to use it. `1` is '-Xlint:all' and `2/3/...` include '-Xdoclint:all'?

Reply:
Seems reasonable. We can change it later if we get complaints.